build.gradle

```
@@ -0,0 +1,19 @@
+plugins {
+    id 'java'
+}
+
+group = 'dev-course'
+version = '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation platform('org.junit:junit-bom:5.9.1')
+    testImplementation 'org.junit.jupiter:junit-jupiter'
```

```suggestion
    testImplementation 'org.junit.jupiter:junit-jupiter:5.7.0'
    testImplementation 'org.assertj:assertj-core:3.18.1'
```

우선 JUnit5 + assertJ 조합으로 테스트 코드를 작성하시는 것을 추천드립니다. 

추후에는 아래와 같은 테스트 라이브러리를 슬쩍 알아보시는 것도 권장합니다.
스프링 프레임워크를 사용할 때 뿐만아니라 순수 자바 코드를 작성할 때도 단위 테스트를 작성할 떄 도움을 주는 프레임워크 입니다.

- [jqwik](https://assertj.github.io/doc/), [Mokito](https://site.mockito.org)